### ISSUE TITLE
asterisk-chan-dongle: empty USSD

### ISSUE THREAD

**Issue by 532910** [openwrt/telephony#687]
Maintainer: @jslachta
Environment: cortexa53 friendlyarm_nanopi-neo2 21.02.0

Description:
Running asterisk as root or chowning `/dev/ttyUSB*` to the `asterisk` user voice calls and SMS become work,
but USSD are just empty:

```
host*CLI> dongle ussd dongle_name *105#
[dongle_name] USSD queued for send
    -- [dongle_name] Successfully sent USSD 0x1df31ae0
NOTICE[4389]: at_response.c:268 at_response_ok: [dongle_name] Successfully sent USSD 0x1df31ae0
 [dongle_name] USSD DCS=2 (0: gsm7, 1: ascii, 2: ucs2)
 [dongle_name] Got USSD type 0 'USSD Notify': ''
```
Compare with 19.07:
```
host*CLI> dongle ussd dongle_name *105#
[dongle_name] USSD queued for send with id 0x36a820a0
    -- [dongle_name] Successfully sent USSD 0x36a820a0
NOTICE[2535]: at_response.c:266 at_response_ok: [dongle_name] Successfully sent USSD 0x36a820a0
 [dongle_name] Got USSD type 0 'USSD Notify': 'Баланс 225.24 pуб.'
```

**Comment by micmac1** [comment]
Can you try upstream master ([link](https://github.com/wdoekes/asterisk-chan-dongle))? If this works we can backport. If not then raising an issue there is probably better. I doubt any chan-dongle hackers read this here. Same for the SMS issue #686 you raised.

**Comment by micmac1** [comment]
Backported fixes to 21.02. Closing issue.

Thanks!
